### PR TITLE
chore: add utility colors

### PR DESCRIPTION
**Pull request by ArianneArcher**
Added Flo brand utility colors

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -61,12 +61,17 @@ $indigo: #6610f2 !default;
 $purple: #6f42c1 !default;
 $pink: #e83e8c !default;
 $red: #ff140f !default;
-$orange: #fd7e14 !default;
+// $orange: #fd7e14 !default;
```
Comment (bexux) [inline scss/_variables.scss]: can we delete these instead of commenting them out?

**Review by bexux — CHANGES_REQUESTED** [review]

File: scss/_variables.scss (hunk 1 above)
Comment (ArianneArcher) [reply scss/_variables.scss]: -No problem

**Review by bexux — APPROVED** [review]
utACK

**Pull request merged**